<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PovRayCameraDialog</class>
 <widget class="QDialog" name="PovRayCameraDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POV-Ray simulated camera</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/oxygen/icons/oxygen/camera-web.png</normaloff>:/oxygen/icons/oxygen/camera-web.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Antialiasing:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="cellSize">
       <property name="suffix">
        <string> µm</string>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>INI file:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cell size:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="antialiasing">
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Camera file:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="focal">
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>8.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="4" colspan="2">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="camLocX">
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="camLocY">
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="camLocZ">
          <property name="suffix">
           <string> m</string>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Camera location:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Focal length:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="5" colspan="2">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="camAxisY">
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="camAxisZ">
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="camAxisX">
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="camRotAngle">
          <property name="suffix">
           <string>°</string>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Camera rotation:&lt;br/&gt;(axis and angle)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Scene file:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Render file:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QPushButton" name="iniFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QPushButton" name="sceneFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="3">
      <widget class="QPushButton" name="cameraFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="3">
      <widget class="QPushButton" name="renderFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camLocX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>locXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camLocY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>locYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camLocZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>locZChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camAxisX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>rotAxisXChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camAxisY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>rotAxisYChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camAxisZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>rotAxisZChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camRotAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>rotationAngleChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>focal</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>focalLengthChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellSize</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>cellSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iniFile</sender>
   <signal>clicked()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>changeIniFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sceneFile</sender>
   <signal>clicked()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>changeSceneFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cameraFile</sender>
   <signal>clicked()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>changeCameraFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renderFile</sender>
   <signal>clicked()</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>changeRenderFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antialiasing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PovRayCameraDialog</receiver>
   <slot>antialiasingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>locXChanged(double)</slot>
  <slot>locYChanged(double)</slot>
  <slot>locZChanged(double)</slot>
  <slot>rotAxisXChanged(double)</slot>
  <slot>rotAxisYChanged(double)</slot>
  <slot>rotAxisZChanged(double)</slot>
  <slot>rotationAngleChanged(double)</slot>
  <slot>focalLengthChanged(double)</slot>
  <slot>cellSizeChanged(double)</slot>
  <slot>antialiasingChanged(int)</slot>
  <slot>changeIniFile()</slot>
  <slot>changeSceneFile()</slot>
  <slot>changeCameraFile()</slot>
  <slot>changeRenderFile()</slot>
 </slots>
</ui>
